src/azure-cli/azure/cli/command_modules/servicefabric/tests/latest/test_sf_managed_application.py

```diff
@@ -235,12 +234,10 @@ def test_managed_service(self):
                          self.check('properties.replicaRestartWaitDuration', '00:11:00')])
         # Update Stateless Service
         self.cmd('az sf managed-service update -g {rg} -c {cluster_name} --application-name {app_name} --service-name {stateless_service_name} '
-                 '--min-instance-count {min_instance_count} --min-instance-percentage {min_instance_percentage} '
-                 '--instance-close-delay-duration {instance_close_delay_duration}',
+                 '--min-instance-count {min_instance_count} --min-instance-percentage {min_instance_percentage} ',
```

Comment:
Should we remove it from command parameters?

Reply:
Model StatelessServiceProperties no longer has parameter instance_close_delay_duration.